operate/schema/src/test/java/io/camunda/operate/schema/IndexSchemaValidatorIT.java

```diff
@@ -80,6 +83,65 @@ public void dropSchema() {
     schemaHelper.dropSchema();
   }
 
+  @Test
+  public void shouldValidateDynamicIndex() {}
```

Comment:
This test case is empty. It should be implemented or removed (I assume the latter).

Reply:
Fixed in the commit Thorben made, this was an oversight on my part from yesterday doing cleanup.